communitiesuk/funding-service-design-post-award-data-frontend
# FMD-218: paketo build

### Change description
Sister PR to: https://github.com/communitiesuk/funding-service-design-post-award-submit/pull/81

Switches image build to paketo buildpack and makes associated changes to static assets build process.



### How to test
Have deployed and tested on dev environment: https://github.com/communitiesuk/funding-service-design-post-award-data-frontend/actions/runs/8017127435/job/21900428844

When run locally CSS and JS should be hot-reloaded. But on deployed environments it will only happen at build time to satisfy contraints of Paketo.


### Screenshots of UI changes (if applicable)
